docs/coordinates/index.rst

```diff
@@ -318,13 +319,21 @@ use this option if you do not need sub-arcsecond accuracy for your coordinates::
 ..
   EXAMPLE END
 
+.. testsetup::
+
+    >>> from astropy.coordinates import EarthLocation, SkyCoord
+    >>> loc = EarthLocation.of_site('Royal Observatory Greenwich', force_builtin=True)
+    >>> keck = EarthLocation.of_site('Keck', force_builtin=True)
```

Comment:
instead of relying on `of_site()` here, you could use it's coordinates hard wired (or append the local `sites.json` on the fly if Keck is missing from it during doctest)

Reply:
Oh yeah, I guess hardcoding lon/lat at `testsetup` would be even better.

Hmmm! This means maybe I don't need to expose `force_builtin` after all...

Reply:
yes, I think if that's an acceptable hack we can avoid both `force_builtin` and the extension of the files. It's a bit of a cheat as the users will see different code than what's tested, but I suppose this one instance it's forgivable.